helmfile/values/dex.yaml.gotmpl

```diff
@@ -29,16 +29,23 @@ config:
     id: google
     name: Google
     config:
-      # Canonical URL of the provider, also used for configuration discovery.
-      # This value MUST match the value returned in the provider config discovery.
-      #
-      # See: https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig
-      issuer: https://accounts.google.com
+      issuer: https://{{ .Values.dex.issuer }}
```

Comment:
I'm not really sure that this needs to be configurable cause there's only one option for it - `https://accounts.google.com`, but sure if you feel like it is needed i'd like you to update our internal helper script https://github.com/elastisys/ck8s/blob/master/helper/configure-apps.bash.

But I would just revert this and and make the okta issuer configurable with like `oktaIssuer` or whatever.

Reply:
ok, will revert that one

Reply:
and will add to changelog